app/src/test/java/com/kickstarter/services/GraphQLTransformersTest.kt

```diff
@@ -176,4 +184,136 @@ class GraphQLTransformersTest : KSRobolectricTestCase() {
         assertTrue(post.publishedAt() == date)
         assertTrue(post.updatedAt() == date)
     }
+
+    @Test
+    fun `test simpleShippingRuleTransformer provides appropriate shippingRule`() {
+        val canadaSimpleSR = FullProject.SimpleShippingRulesExpanded(
+            "SimpleShippingRule",
+            "17.34562379823645234875620384756203847234",
```

Comment:
This string value will be converted to a double by the `simpleShippingRuleTransformer` on line 220. 
The obtained double value can be checked on line 227